src/main/java/frc/robot/commands/ActuatorDown.java

```diff
@@ -39,6 +47,6 @@ public void end(boolean interrupted) {
   // Returns true when the command should end.
   @Override
   public boolean isFinished() {
-    return down = true;
+    return RobotContainer.m_intakeSubsystem.isDown() == true;
```

Comment:
Why is this referring to RobotContainer.m_intakeSubsystem instead of the commands m_intakeSubsystem (which was passed in from RobotContainer)?